cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/FileLocalizer.java

```diff
@@ -40,22 +41,31 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.nio.file.StandardCopyOption;
+import java.util.Random;
+import java.util.concurrent.Callable;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipInputStream;
+import javax.naming.ServiceUnavailableException;
```

Comment:
I assume you meant to use the cdap one? Though that is in cdap-common which shouldn't be a dependency here.

You can also catch RetryableException, which is in cdap-api.

Reply:
Thanks, fixed.